sa/database.go

```diff
@@ -182,3 +183,10 @@ func initTables(dbMap *gorp.DbMap) {
 	dbMap.AddTableWithName(core.SignedCertificateTimestamp{}, "sctReceipts").SetKeys(true, "ID").SetVersionCol("LockCol")
 	dbMap.AddTableWithName(core.FQDNSet{}, "fqdnSets").SetKeys(true, "ID")
 }
+
+func safeSelectOne(m *gorp.DbMap, holder interface{}, q string, args ...interface{}) error {
+	if strings.ContainsAny(q, "'\"\n\r\\") {
```

Comment:
This character set is defined in the mysql method `mysql_real_escape_string` [here](https://dev.mysql.com/doc/refman/5.7/en/mysql-real-escape-string.html).

I think we are fine just checking that there are no values that would need to be encoded since we are only ever inserting string literals from source. Given my limited research Gorp only supports sending multiple statements using `gorp.Exec`, otherwise it fails out with a syntax error.